src/widgets/refinement-list/refinement-list.js

```diff
@@ -66,7 +66,7 @@ const renderer = ({
       showMore={showMoreConfig !== null}
       templateProps={renderState.templateProps}
       toggleRefinement={refine}
-      searchFacetValues={searchForFacetValues && searchForItems}
+      searchFacetValues={searchForFacetValues ? searchForItems : () => {}}
```

Comment:
I think the least impactful is not to change the RefinementList, update the propType and use undefined. If only we had option types 🤓